docs/system/UTILS.stories.mdx

```diff
@@ -105,8 +112,11 @@ It gets 2 params:
 2. shade
 
 And returns the AA color for that.
-   <br />
-   <br />
+
+{' '}
```

Comment:
remove this

Reply:
fixed on [`b7f1359` (#602)](https://github.com/Orfium/orfium-ictinus/pull/602/commits/b7f1359dc90fbc564437a368355fb1a515c28fd4)